src/Push.js

```diff
@@ -185,6 +191,11 @@ class Push extends PushBase {
 				newConfig.service = settings.data[newConfig.serviceName];
 			newConfig.serviceSettingsHash = settings.hash;
 
+			// Expand environment variables
```

Comment:
@ephemerant Can you elaborate on this addition please and explain how it relates to Windows and/or vscode?

Reply:
It's so I can, for example, refer to the root as "%userprofile%\\desktop\\sync", allowing for it to expand to "C:\\users\\ephemerant\\desktop\\sync".

See:
- https://superuser.com/a/217506
- https://stackoverflow.com/a/21363956